### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by zAuk000** [cassiePython/NeRF-Art#6]
Hi，
will you release the training code soon? or could I request a copy of the implement of the criteria ？thanks！

**Comment by cassiePython** [comment]
You can send me an email. I will share you with a copy of the training code and loss functions.

**Comment by zAuk000** [comment]
> You can send me an email. I will share you with a copy of the training code and loss functions.

I have sent an email by 15634103353@163.com，looking forward to your share！
